src/main/kotlin/com/breed/app/model/DogBreed.kt

```diff
@@ -0,0 +1,18 @@
+package com.breed.app.model
+
+import org.springframework.data.relational.core.mapping.Table
+
+import org.springframework.data.annotation.Id
+
+@Table("dog_breed")
+data class DogBreed(
```

Comment:
please double-check whether its OK to use data classes as Hibernate entities (especially when we`re using r2dbc)